x-pack/plugins/cloud_security_posture/public/pages/configurations/utils/create_detection_rule_from_benchmark.ts

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
@animehart @opauloh This file is almost identical to `create_detection_rule_from_finding.ts`. Let's avoid such repetition and think about how to abstract the logic. I can see that the only different thing is where we take the data from. In one case it's `finding.rule.benchmark` and in another, it's `benchmarkRule`. All the rest look the same. If that's the case, it should be very simple to abstract and just pass the `benchmark` object